libr/bin/open.c

```diff
@@ -20,17 +20,3 @@ R_API int r_bin_open(RBin *bin, const char *filename, RBinOptions *bo) {
 	}
 	return -1;
 }
-
-R_API RBinFile *r_bin_get_file(RBin *bin, int bd) {
-	return r_id_storage_take (bin->ids, bd);
-}
-
-R_API bool r_bin_close(RBin *bin, int bd) {
```

Comment:
uhm i think we dont want to remove this API because the idea was to make all rbinobj apis use the idstorage instead of returning pointers like condret did for siol.

before removing this i would like to discuss about the api design

Reply:
if you keep r_bin_open dont kill r_bin_close

Reply:
ok